my-app/src/App.js

```diff
@@ -1,41 +1,29 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar } from "./frontend/components/navbar/Navbar";
+import { Footer } from "./frontend/components/footer/Footer";
+import {
+  Home,
+  History,
+  Liked,
+  Playlist,
+  WatchLater,
+} from "./frontend/pages/index";
+import { Routes, Route } from "react-router-dom";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Navbar />
+
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/history" element={<History />} />
+        <Route path="/liked" element={<Liked />} />
+        <Route path="/playlist" element={<Playlist />} />
+        <Route path="/watchLater" element={<WatchLater />} />
+      </Routes>
+
+      <Footer />
```

Comment:
you can have all your routes in a different file.

Reply:
I think there is no need of this.